draft-ietf-quic-transport.md

```diff
@@ -5284,7 +5284,7 @@ ACK Delay:
   to the power of the value of the ack_delay_exponent transport parameter set by
   the sender of the ACK frame; see {{transport-parameter-definitions}}.  Scaling
   in this fashion allows for a larger range of values with a shorter encoding at
-  the cost of lower resolution.  Because the receiver doesn't use the ACK Delay
+  the cost of lower resolution.  Because the receiver doesn't use the ACK delay
   for Initial and Handshake packets, a sender SHOULD send a value of 0.
```

Comment:
This refers to the field, so it should be "ACK Delay field" again.

Reply:
OK